src/Microsoft.AspNetCore.ResponseCaching/ResponseCacheFeature.cs

```diff
@@ -6,8 +6,8 @@
 namespace Microsoft.AspNetCore.ResponseCaching
 {
     // TODO: Temporary interface for endpoints to specify options for response caching
-    public class ResponseCachingFeature
+    public class ResponseCacheFeature
```

Comment:
This is the feature for apps to set custom vary-by's. It has to be a feature because there's no header for this one.